kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/OperationSupport.java

```diff
@@ -617,8 +617,8 @@ public static Status createStatus(HttpResponse<?> response, KubernetesSerializat
             return status;
           }
         }
-      } catch (IOException | KubernetesClientException | IllegalArgumentException e) {
-        // ignored
+      } catch (IOException | RuntimeException e) {
```

Comment:
Shall we just capture `Exception` instead?

I'm not sure which kind of exception might fall into the basket of  non-caught exceptions in this case-

Reply:
> Shall we just capture Exception instead?

I'm minimally trying to be complaint with the idea of not simply catching Exception - but RuntimeException isn't much better.

> I'm not sure which kind of exception might fall into the basket of non-caught exceptions in this case-

Nothing, IOException is the only checked exception - the method doen't throw anything.